data/human/transport missions.txt

```diff
@@ -3209,3 +3209,76 @@ mission "Earth Retirement"
 			label reject
 			`	Charles looks surprised for a moment, and then, for the first time you've seen, he smiles. "Thank you, Captain <last>. I wish there were more people like you in the Paradise Worlds."`
 			`	You wish the couple the best of luck on Earth, and they slowly make their way to the train station. Presumably, their new apartment is far from here.`
+
+
+
+mission "Spacediving professionals"
+	name "Spacediving at <destination>"
+	description "Drop skydivers from the edge of space. They are carrying cooled spacesuits, durable parachutes and oxygen bottles to ensure the survival of their party. Payment is <payment>."
+	passengers 5 10
+	repeat
+	job
+	source
+		government "Republic" "Free Worlds" "Syndicate"
+	destination
+		distance 7 9
+		government "Republic" "Free Worlds" "Syndicate"
+	to offer
+		and
+			credits >= 1000000
+			random > 96
```

Comment:
Why does the player need 1 million credits to have this offered?
```suggestion
		credits >= 1000000
		random < 3
```

Reply:
I reduced the credits requirement and only placed it on the job with the fine.